CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#837](https://github.com/nix-rust/nix/pull/837))
 
 ### Fixed
+- Don't derive Clone for sys::aio::buffer
```

Comment:
I'd put this under "Removed" since this actually removed something from the API.